app/models/publisher_status_update.rb

```diff
@@ -0,0 +1,11 @@
+class PublisherStatusUpdate < ApplicationRecord
+  has_paper_trail
```

Comment:
Make sure you delete this line. This model essentially is its own paper_trail.

I realize it seems silly to run our own paper trail, but essentially we don't want to bloat 1 table too hard to affect the rest of the database which depends on a `versions` table.